src/pages/Main/Main.js

```diff
@@ -1,7 +1,67 @@
-import React from 'react';
+import Card from 'components/Card/Card';
+import React, { useEffect, useRef, useState } from 'react';
+import styled from 'styled-components';
 
 const Main = () => {
-  return <h1>Abnb 메인 페이지 입니다</h1>;
+  const [itemList, setItemList] = useState([]);
+  const observerTarget = useRef(null);
+  //TODO : Back API 구현 후 추가 구현 예정
+  useEffect(() => {
+    const observer = new IntersectionObserver(entries => {
+      if (entries[0].isIntersecting) {
+        console.log('됐어!');
+        fetch('/data/MOCK.json')
+          .then(res => res.json())
+          .then(data => setItemList(prev => [...prev, ...data]));
+      }
+    });
+    if (observerTarget.current) {
+      observer.observe(observerTarget.current);
+    }
+    return () => {
+      observer.disconnect();
+    };
+  }, []);
+
+  // const loadMore = () => {
+  //   setPage((prev) => prev + 1);
+  // };
+
+  return (
+    <>
+      <ProductListContainer>
+        <ProductListBox>
+          {itemList.map(data => (
+            <Card key={data.id} {...data} />
+          ))}
+        </ProductListBox>
+      </ProductListContainer>
+      <div ref={observerTarget} />
+    </>
+  );
 };
 
 export default Main;
+
+const ProductListContainer = styled.div`
+  width: fit-content;
+  margin: 20px auto;
+`;
+
+const ProductListBox = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
```

Comment:
how about
```suggestion
  grid-template-columns: repeat(2, 1fr);
```
css의 repeat 함수를 사용하는 방식입니다!